actionpack/test/dispatch/runner_test.rb

```diff
@@ -0,0 +1,18 @@
+require "abstract_unit"
+
+class RunnerTest < ActiveSupport::TestCase
+  test "runner preserves the setting of integration_session" do
+    runner = Class.new do
+      def before_setup
```

Comment:
calling `super` here will make the test fail saying that there is no super class method `before_setup`.